src/style.css

```diff
@@ -1,3 +1,62 @@
 body {
   margin: 0;
+
+  --blue-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.header__logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  margin: 0;
```

Comment:
I always recommend to reset css with [this](https://meyerweb.com/eric/tools/css/reset/) stylesheet